src/Altinn.AccessManagement.Persistence/DelegationMetadataRepo.cs

```diff
@@ -209,9 +209,9 @@ private async Task<DelegationChange> InsertAppDelegation(DelegationChange delega
                 { "delegationChangeType", delegationChange.DelegationChangeType },
                 { "altinnAppId", delegationChange.ResourceId },
                 { "offeredByPartyId", delegationChange.OfferedByPartyId },
-                { "coveredByUserId", delegationChange.CoveredByUserId.HasValue ? delegationChange.CoveredByUserId.Value : DBNull.Value },
-                { "coveredByPartyId", delegationChange.CoveredByPartyId.HasValue ? delegationChange.CoveredByPartyId.Value : DBNull.Value },
-                { "performedByUserId", delegationChange.PerformedByUserId.HasValue ? delegationChange.PerformedByUserId.Value : DBNull.Value },
+                { "coveredByUserId", delegationChange.CoveredByUserId },
```

Comment:
Either CoveredByUserId or CoveredByPartyId will be 0 which should not be written to the database. 
If Dapper don't support the passing of DBNull we might need checks for which params should be added.

Reply:
It supports nullable values, so we do not need to send DbNull.Value. If CoveredByPartyId is null then it will be NULL in the database. Code is working as before.